lms/lmsdb/models.py

```diff
@@ -298,15 +344,38 @@ class Exercise(BaseModel):
     due_date = DateTimeField(null=True)
     notebook_num = IntegerField(default=0)
     order = IntegerField(default=0, index=True)
+    course = ForeignKeyField(Course, backref='exercise')
+    number = IntegerField()
```

Comment:
These should be in the migrations

Reply:
Because these fields aren't nullable, what do you think about this kind of migration:
- Creating a `Course` - (I need more info about it like the `name` of it and `date`).
- Creating  `UserCourse`s objects for all users in the db and connect them to the `Course` we've created.
- Adding the `Course` to this `course` column.
- Adding the exercise `id` to the exercise `number`.

Reply:
We might want to automatically fill this value according to the last number for this course in the database